packs/trendmicro-cloudone-conformity/config.yml

```diff
@@ -0,0 +1,34 @@
+
+# Name of the pack (required)
+name: trendmicro-cloudone-conformity
+
+title: New Relic’s integration with Trend Micro Cloud One - Conformity
```

Comment:
is the `-` needed? can this be  `New Relic’s integration with Trend Micro Cloud One Conformity`

Reply:
same question applies to all the uses of the name with the `-`

Reply:
Yes, we need the "-". "Cloud One" is the product and "Conformity" is one of the services it offers. Please see this https://www.trendmicro.com/content/dam/trendmicro/global/en/business/products/hybrid-cloud/cloud-one/conformity/ds_cloud_one_conformity.pdf

Reply:
ok. thanks for clarifying!